apis/capabilities/v1beta1/product_types.go

```diff
@@ -353,8 +353,12 @@ type OIDCSpec struct {
 	IssuerType string `json:"issuerType"`
 
 	// Issuer is the OIDC issuer
+	// +optional
 	IssuerEndpoint string `json:"issuerEndpoint"`
 
+	// IssuerEndpointRef  is the reference to OIDC issuer Secret that contains IssuerEndpoint
+	IssuerEndpointRef corev1.SecretReference `json:"issuerEndpointRef,omitempty"`
```

Comment:
Just a reminder, with this new field in the product CRD, anyone could create a Product CR with the `issuerEndpointRef` filled and with the current implementation, it would not work,

Reply:
done. Thank you